src/formatDate.js

```diff
@@ -50,7 +50,50 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat[3];
+  const newSeparator = toFormat[3];
+
+  const fromArr = date.split(oldSeparator);
+  const lastIndex = toFormat.length - 1;
+
+  const oldDay = fromArr[fromFormat.findIndex((item) => item.includes('D'))];
+  const oldMonth = fromArr[fromFormat.findIndex((item) => item.includes('M'))];
+  const oldYear = fromArr[fromFormat.findIndex((item) => item.includes('Y'))];
```

Comment:
```suggestion
  const getItem = mask => fromArr[fromFormat.findIndex((item) => item.includes(mask))]
  const oldDay = getItem('D');
  const oldMonth = getItem('M');
  const oldYear = getItem('Y');
```
Don't repeat yourself.